amazon/iontest/ion_test_driver.py

```diff
@@ -96,7 +96,7 @@ def check_tool_dependencies(args):
             # to call a tool-specific command to test the existence of the executable. This should be a command that
             # always returns zero.
```

Comment:
This comment should be removed now that you've implemented the generalized logic.